docs/src/pages/demos/dialogs/dialogs.md

```diff
@@ -42,7 +42,7 @@ If a title is required:
 
 You can also swap out the transition, the next example uses `Slide`.
 
-{{"demo": "pages/demos/dialogs/AlertDialogSlide.js"}}
+{{"demo": "pages/demos/dialogs/AlertDialogSlide.js", "hideEditButton": true}}
```

Comment:
This is because of a temporary workaround. Please keep it live.
https://github.com/mui-org/material-ui/blob/f46b77827cbecad926eb113bad0fb6a695f94e40/docs/src/pages/demos/dialogs/AlertDialogSlide.js#L10

Reply:
Ok, sorry didn't see comment!